pymc3/math.py

```diff
@@ -73,6 +73,50 @@ def __str__(self):
 
 def probit(p):
     return -sqrt(2) * erfcinv(2 * p)
-    
+
 def invprobit(x):
     return 0.5 * erfc(-x / sqrt(2))
+
+
+def expand_packed_triangular(n, packed, lower=False, diagonal_only=False):
+    """Convert a packed triangular matrix into a two dimensional array.
```

Comment:
Thanks. Do you think we should take `n` (the size of the matrix) as argument explicitly? We could infer it from the shape of `packed`, but this involves integer square roots, and I don't know how to do that safely in theano (or throw a useful error if it does not work).